packages/azure-storage/files/helm-chart/templates/NFS.yaml

```
@@ -0,0 +1,47 @@
+---
+apiVersion: storage.azure.upbound.io/v1beta1
+kind: Account
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/part-of: {{ .Values.organization }}.{{ .Values.project }}
+    app.kubernetes.io/managed-by: kubepak
+  name: {{ .Values.organization }}{{ .Values.project }}nfs
+spec:
+  forProvider:
+    accountKind: "BlockBlobStorage"
+    accountTier: "Premium"
+    accountReplicationType: ZRS
```

we can move to `GRS` or `GZRS` in production. During our testing, we can use a simple `LRS`
```suggestion
    accountReplicationType: "{{ index .Values "packages" "azure-storage" "nfs" "accountReplicationType"}}"
```